JiraTogglSync.CommandLine/Program.cs

```diff
@@ -9,8 +9,21 @@ class Program
 	{
 		static void Main(string[] args)
 		{
-			var togglApiKey = ConfigurationHelper.GetEncryptedValueFromConfig("toggl-api-key", () => AskFor("Toggl API Key"));
-			var toggl = new TogglService(togglApiKey);
+		    var purge = false; //for now purge is not supported
```

Comment:
No dead code; if this is not needed, it should not be included in this PR (same with line 19)